package.json

```diff
@@ -26,16 +26,19 @@
     "@sinonjs/fake-timers": "^8.0.0",
     "autocannon": "^7.5.0",
     "concurrently": "^6.3.0",
-    "fastify": "^3.21.3",
-    "mercurius": "^8.2.1",
+    "fastify": "^3.24.0",
+    "mercurius": "^8.9.1",
     "snazzy": "^9.0.0",
     "standard": "^16.0.3",
     "tap": "^15.0.9",
     "wait-on": "^6.0.0"
   },
   "dependencies": {
+    "ajv": "^6.12.6",
     "async-cache-dedupe": "^0.5.0",
-    "fastify-plugin": "^3.0.0"
+    "fastify-plugin": "^3.0.0",
+    "fluent-json-schema": "^3.0.1",
+    "mercurius-cache": "^0.10.0"
```

Comment:
why this package?

Reply:
Not sure I get the question. can you be a bit more specific?

Reply:
why is this package installing itself as a dependency?

Reply:
never mind 😄 you mean the mercurius-cache, right? It was added when I ran the start command in the read me file. removing it now

Reply:
the instructions in the readme are for _users_ of this package, as opposed to developers working on this package